.github/workflows/sanity.yml

```diff
@@ -0,0 +1,76 @@
+name: Sanity
+
+on:
+  pull_request:
+    paths:
+      - "data/**"
+      - "python/**"
+
+jobs:
+  main:
+    runs-on: "ubuntu-latest"
+    container:
+      image: "quay.io/centos/centos:stream9"
+
+    steps:
+      - name: "Checkout the repository"
+        uses: actions/checkout@v4
```

Comment:
run this after you install git-core (ie the step after this), otherwise there is the risk that the checked out repo is not a proper git repo